Multisig/UI/Onboarding/LaunchView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  LaunchView.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 27.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+extension HorizontalAlignment {
+    private enum CenterHorizontalAlignment : AlignmentID {
+        static func defaultValue(in d: ViewDimensions) -> CGFloat {
+            return d[HorizontalAlignment.center]
+        }
+    }
+
+    static let centerHorizontalAlignment = HorizontalAlignment(CenterHorizontalAlignment.self)
+}
+
+extension VerticalAlignment {
+    private enum CenterVerticalAlignment : AlignmentID {
+        static func defaultValue(in d: ViewDimensions) -> CGFloat {
+            return d[VerticalAlignment.center]
+        }
+    }
+
+    static let centerVerticalAlignment = VerticalAlignment(CenterVerticalAlignment.self)
+}
+
+extension Alignment {
+    static let centerAlignment = Alignment(horizontal: .centerHorizontalAlignment, vertical: .centerVerticalAlignment)
+}
+
+struct LaunchView: View {
+    @Binding var acceptedTerms: Bool
+    @State var isAgreeWithTermsPresented = false
+
+    var body: some View {
+        GeometryReader { gp in
+            ZStack(alignment: .centerAlignment) {
+                // anchor to position text image in the center of the screen
+                Rectangle()
+                    .frame(width: 0, height: 0)
+                    .alignmentGuide(.centerVerticalAlignment) { d in d[VerticalAlignment.center] }
+                    .position(y: gp.size.height / 2)
+
+                VStack(alignment: .center, spacing: 40) {
+                    Image("launchscreen-logo") // 100 x 153 px
```

Comment:
I don't understand why this comment is needed here

Reply:
Potentially the image might be larger than the smallest supported screen size. So with this, I indicated why I don't apply any additional frames.

Reply:
can you add it with words, otherwise this cannot be understood

Reply:
yes, I will adjust wording.